xbmc/utils/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES ActorProtocol.cpp
             AlarmClock.cpp
             AliasShortcutUtils.cpp
+            AMLUtils.cpp
```

Comment:
I have force pushed, yes. Either if AML is defined or if we are on android in general.